hedera-node/hedera-mono-service/src/main/java/module-info.java

```diff
@@ -235,6 +235,8 @@
     opens com.hedera.node.app.service.mono.state.migration to
             com.swirlds.common;
 
+    exports com.hedera.node.app.service.mono.fees.calculation.meta;
```

Comment:
Do we need this export ?

Reply:
Yes, it's used in NetworkGetVersionInfoHandler for FixedUsageEstimates import